example.toml

```diff
@@ -75,7 +75,8 @@ start = "2024-01-04T11:00:00.00Z"
 # If `start` is a date and time then `end` must be too, and vice versa when `start` is
 # just a date then `end` must be too.
 # If not provided and `start` is just a date (with no time component), then this is an all-day
-# event on the `start` date. Must be provided if `start` is a date and time.
+# event on the `start` date. Must be provided if `start` is a date and time. `start` and `end` must
+# be in the same timezone.
```

Comment:
This doesn't seem right...

Reply:
Fixed - should only apply for `recurrence_id`.